test/e2e/annotations/canary.go

```diff
@@ -806,7 +805,7 @@ var _ = framework.DescribeAnnotation("canary-*", func() {
 				Contains(canaryService)
 		})
 
-		ginkgo.It("should route requests evenly split between mainline and canary if canary weight is 50", func() {
+		ginkgo.It("should route requests splitted between mainline and canary if canary weight is 50", func() {
```

Comment:
https://onlymyenglish.com/split-past-tense/

Reply:
whoops, thanks for catching that, will fix it...

Reply:
No worries. Please allow some time for me to digest what your change here is about. The grammar nit was just something that jumped at me. :)